site/src/@ndlib/gatsby-theme-marble/components/Pages/BrowsePage/index.js

```diff
@@ -50,36 +50,36 @@ const Browse = ({ location }) => {
   const formats = [
     {
       label: 'Paintings',
-      target: '/search?format[0]=Paintings',
+      target: '/search?format[0]=paintings',
     },
     {
       label: 'Maps',
       target: '/search?format[0]=Cartographic%20material',
     },
     {
       label: 'Photographs',
-      target: '/search?format[0]=Photographs',
+      target: '/search?format[0]=photographs',
     },
     {
       label: 'Prints and Posters',
-      target: '/search?format[0]=Prints&format[1]=Two-dimensional%20nonprojected%20graphic&format[2]=Projected%20medium',
+      target: '/search?format[0]=prints&format[1]=Two-dimensional%20nonprojected%20graphic&format[2]=Projected%20medium',
     },
     {
       label: 'Drawings',
-      target: '/search?format[0]=Drawings',
+      target: '/search?format[0]=drawings',
     },
     {
       label: 'Sculpture',
-      target: '/search?format[0]=Sculpture',
+      target: '/search?format[0]=sculpture',
     },
     {
       label: 'Ceremonial objects and regalia',
-      target: '/search?format[0]=Ceremonial%20objects%20and%20regalia',
+      target: '/search?format[0]=ceremonial%20objects%20and%20regalia',
 
     },
     {
       label: 'Decorative Arts, Craft, and Design',
-      target: '/search?format[0]=Decorative%20Arts%2C%20Craft%2C%20and%20Design',
+      target: '/search?format[0]=ceramics&format[1]=glass&format[2]=metalwork',
     },
     {
```

Comment:
Do "Musical%20sound%20recording&format[1]=Notated%20music" and "Language material"  also need lowercased?